<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="serverTabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Account">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="logInBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>301</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Welcome!</string>
      </property>
      <widget class="QLineEdit" name="usernameField">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>91</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="username_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLabel" name="title_label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Admin Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="password_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="adminSignInButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordField">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="adminFunctions">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>281</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="loggedInAsLayout">
       <item>
        <widget class="QLabel" name="loggedInAs">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Logged in as:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="usernameLabel">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>1011</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="controlLayout">
       <item>
        <widget class="QGroupBox" name="serverFunctions">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>190</y>
            <width>301</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="serverControlLayout">
           <item>
            <widget class="QPushButton" name="startServerButton">
             <property name="text">
              <string>Start Server</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopServerButton">
             <property name="text">
              <string>Stop Server</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>150</y>
            <width>301</width>
            <height>50</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="ServerControlLayout">
           <item>
            <widget class="QLabel" name="ServerStatusLabel">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Server Status: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="StatusLabel">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: red;</string>
             </property>
             <property name="text">
              <string>Not Running</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="addNewAdmin">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>70</y>
            <width>421</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="2">
              <widget class="QLineEdit" name="fNameField"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="emailLabel">
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="fNameLabel">
               <property name="text">
                <string>First Name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="emailField"/>
             </item>
             <item row="0" column="4">
              <widget class="QLineEdit" name="lNameField"/>
             </item>
             <item row="1" column="4">
              <widget class="QLineEdit" name="passField"/>
             </item>
             <item row="1" column="3">
              <widget class="QLabel" name="passLabel">
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="lNameLabel">
               <property name="text">
                <string>Last Name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="createAccountButton">
             <property name="text">
              <string>Create Account</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="createAdminLabel">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>20</y>
            <width>171</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>Create Administrator Account</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="UserStats">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="activeUsers">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>461</width>
        <height>481</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="activeUsersLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Active Users</string>
      </property>
     </widget>
     <widget class="QLabel" name="userListLabel">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>User List</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="userInfo">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>461</width>
        <height>481</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="RepoStats">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="activeReposLabel">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>410</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Active Repositories</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="activeReposTable">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>370</y>
        <width>51</width>
        <height>81</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Repository Name</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>User</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="repositoryComboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="repoTitleLabe">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>View Repository Details:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="repoInfoTable">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>361</width>
        <height>441</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>User</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Permission</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="addContributorBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add a Contributor:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="addContribPerm">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Read</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Read &amp; Write</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Execute</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Permission</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="addContribEmail">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addContribButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>221</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Contributor</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="remContributorBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Remove a Contributor</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reason</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="remContribEmail">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QPushButton" name="remContribButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>221</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove Contributor</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="remContribReason">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="isPrivateLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="createRepoBox">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>120</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="createRepoButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>231</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Repository</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="publicCheckBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>78</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Public</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create New Repository</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="privateCheckBox">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Private</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="RepoName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>171</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
